front/src/Api/iframe/Area/Area.ts

```diff
@@ -1,4 +1,4 @@
-import { ITiledMapObject } from "../../../Phaser/Map/ITiledMap";
+import { ITiledMapObject } from "@workadventure/tiled-map-type-guard/dist";
```

Comment:
Question: do you think we can get rid of the "dist" part in the URL (by editing "@workadventure/tiled-map-type-guard" of course). It looks to me the package is not properly designed (my very fault)